.buildkite/steps/cleanup.sh

```diff
@@ -4,8 +4,7 @@ set -uxo pipefail
 
 ########################################################
 # We make an attempt to clean up the specific resources created during this pipeline elsewhere. However
-# sometimes that fails, are various CloudFormation Stacks, S3 Buckets and CloudWatch Log Groups can be
-# left around.
+# sometimes that fails, are various CloudFormation Stacks and S3 Buckets left around.
 #
 # This is a safety net at the end of the build that attempts to delete any resources created by this
 # pipeline more than a day ago, regardless of if they were created in the current build or not.
```

Comment:
Are we making any changes to this cleanup process, or just updating the comment?

Reply:
I think they're referring to logs created during the AMI build, which I'm hoping won't be created, but I may be wrong. I was mostly trying to keep comments current with the changes we're making. My plan is to delete all log groups from CloudWatch before deploying these changes and check if any of them are recreated.